src/domain.ts

```diff
@@ -103,6 +116,7 @@ interface Editor {
   showError(message: string): void;
   showMessage(message: string): void;
   getConfiguration(): Configuration;
+  slidesRcExist(): boolean;
```

Comment:
I think this is implementation detail of the VS Code adapter. It shouldn't be part of the Domain interface.

That's fine, I'll simply refactor this 👍